### ISSUE TITLE
Immediate crash on check and lint

### ISSUE THREAD

**Issue by css-optivoy** [biomejs/biome#10864]
### Environment information

<details>

```bash

CLI:
  Version:                      2.5.2
  Color support:                true

Platform:
  CPU Architecture:             x86_64
  OS:                           linux

Environment:
  BIOME_DISTRIBUTION:                unset
  BIOME_LOG_PATH:                    unset
  BIOME_LOG_PREFIX_NAME:             unset
  BIOME_LOG_LEVEL:                   unset
  BIOME_LOG_KIND:                    unset
  BIOME_CONFIG_PATH:                 unset
  BIOME_THREADS:                     unset
  BIOME_WATCHER_KIND:                unset
  BIOME_WATCHER_POLLING_INTERVAL:    unset
  NO_COLOR:                     unset
  TERM:                         xterm-256color
  JS_RUNTIME_VERSION:           unset
  JS_RUNTIME_NAME:              unset
  NODE_PACKAGE_MANAGER:         unset

Biome Configuration:
  Status:                       Not set
  Path:                         unset

Workspace:
  Open Documents:               0

```

</details>

### What happened?

I ran `biome lint ./my.html` and `biome check ./my.html` and both immediately crashed:

```bash

$ RUST_BACKTRACE=1 biome check ./dist/bundle.html
Biome encountered an unexpected error

This is a bug in Biome, not an error in your code, and we would appreciate it if you could report it to https://github.com/biomejs/biome/issues/ along with the following information to help us fixing the issue.

When opening the issue, please provide a minimal reproduction, or identify and share the file/code that triggers it. Without a way to reproduce the error, the error can't be fixed:

Source Location: crates/biome_text_size/src/range.rs:73:9
Thread Name: biome::workspace_worker_0
Message: assertion failed: start.raw <= end.raw
Stack Trace:
   0: <unknown>
   1: <unknown>
   2: <unknown>
   3: <unknown>
   4: <unknown>
   5: <unknown>
   6: <unknown>
   7: <unknown>
   8: <unknown>
   9: <unknown>
  10: <unknown>
  11: <unknown>
  12: <unknown>
  13: <unknown>
  14: <unknown>
  15: <unknown>
  16: <unknown>
  17: <unknown>
  18: <unknown>
  19: <unknown>
  20: <unknown>
  21: <unknown>
  22: <unknown>
  23: <unknown>
  24: <unknown>
  25: <unknown>
  26: start_thread
             at ./nptl/pthread_create.c:454:8
  27: __GI___clone3
             at ./misc/../sysdeps/unix/sysv/linux/x86_64/clone3.S:78:0


dist/bundle.html internalError/panic  INTERNAL  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ✖ processing panicked: assertion failed: start.raw <= end.raw
  
  ⚠ This diagnostic was derived from an internal Biome error. Potential bug, please report it if necessary.
  

Checked 0 files in 54ms. No fixes applied.
check ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ✖ No files were processed in the specified paths.
  
  ℹ Check your biome.json or biome.jsonc to ensure the paths are not ignored by the configuration.
  
  ℹ These paths were provided but ignored:
  
  - ./dist/bundle.html
```

### Expected result

No crash.

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by ematipico** [comment]
> When opening the issue, please provide a minimal reproduction, or identify and share the file/code that triggers it. Without a way to reproduce the error, the error can't be fixed:

**Comment by biome-cookie** [comment]
Hello @css-optivoy, please provide a minimal reproduction. You can use one of the following options:

- Provide a link to [our playground](https://biomejs.dev/playground), if it's applicable.
- Provide a link to GitHub repository. To easily create a reproduction, you can use our interactive CLI via `npm create @biomejs/biome-reproduction`

Issues marked with `S-Needs repro` will be **closed** if they have **no activity within 3 days**.

**Comment by css-optivoy** [comment]
I can't share the original file, but after a bit of trial-and-error I've managed to cut it down to this, that triggers the same fault:
```html
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>title</title>
  </head>
  <body>
    <textarea rows=4></textarea>
  </body>
</html>
```

Specifically the `rows=4` on `textarea` seems to be the trigger, for whatever reason.

**Comment by siketyan** [comment]
Reproducible on the playground: https://biomejs.dev/playground/?lintRules=all&tab=formatter&pane=Diagnostics&code=PAAhAEQATwBDAFQAWQBQAEUAIABoAHQAbQBsAD4ACgA8AGgAdABtAGwAIABsAGEAbgBnAD0AIgBlAG4AIgA%2BAAoAIAAgADwAaABlAGEAZAA%2BAAoAIAAgACAAIAA8AG0AZQB0AGEAIABjAGgAYQByAHMAZQB0AD0AIgB1AHQAZgAtADgAIgA%2BAAoAIAAgACAAIAA8AHQAaQB0AGwAZQA%2BAHQAaQB0AGwAZQA8AC8AdABpAHQAbABlAD4ACgAgACAAPAAvAGgAZQBhAGQAPgAKACAAIAA8AGIAbwBkAHkAPgAKACAAIAAgACAAPAB0AGUAeAB0AGEAcgBlAGEAIAByAG8AdwBzAD0ANAA%2BADwALwB0AGUAeAB0AGEAcgBlAGEAPgAKACAAIAA8AC8AYgBvAGQAeQA%2BAAoAPAAvAGgAdABtAGwAPgA%3D&language=html

**Comment by dyc3** [comment]
Its the `rows=4` that's causing the bug, more specifically it being unquoted causes the panic.
